DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Reports/ReportsViewModel.cs

```diff
@@ -42,4 +45,14 @@ public ActivityDataRowModel(PeriodOfActivity periodOfActivity, string format)
         public int Evaluations { get; set; }
         public int Registrations { get; set; }
     }
+
+    public class ActivityFilterModel
+    {
+        public string? JobGroupName { get; set; }
+        public string? CourseCategoryName { get; set; }
+        public string? CustomisationName { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+        public string? ReportIntervalName { get; set; }

```

Comment:
Interval and job group name should be not nullable as they have defaults (it's "All" for job group).